index.js

```diff
@@ -8,6 +8,71 @@ const checkWIPModule = require('./lib/checkWipDraftPR');
 
 const whitelistedAccounts = (
   (process.env.WHITELISTED_ACCOUNTS || '').toLowerCase().split(','));
+const checksWhitelist = {
+  'oppia-android': {
+    'opened': ['cla-check'],
+    'reopened': [],
+    'labeled': [],
+    'synchronize': [],
+    'closed': [],
+    'edited': []
+  },
+  'oppia': {
+    'opened': [
+      'cla-check',
+      'changelog-check',
+      'branch-check',
+      'wip-check'],
+    'reopened': [
+      'changelog-check',
+      // Prevent user from reopening the PR.
+      'branch-check',
+      'wip-check'
+    ],
+    'labeled': ['assignee-check',],

```

Comment:
Nit: no terminating ``,`` in an array list in JavaScript.

Reply:
Done.